src/components/StorageOverview/Inventory/Inventory.js

```diff
@@ -10,24 +10,28 @@ import {
 } from '../../Dashboard/DashboardCard';
 import { StorageOverviewContext } from '../StorageOverviewContext';
 import { mapNodesToProps, mapPvcsToProps, mapPvsToProps } from '../../Dashboard/Inventory/utils';
+import diskStatsToProps from './diskInventoryUtils';
 import { InventoryRow } from '../../Dashboard/Inventory/InventoryRow';
 
-const InventoryBody = ({ nodes, pvs, pvcs }) => (
+const InventoryBody = ({ nodes, pvs, pvcs, disks }) => (
   <React.Fragment>
     <InventoryRow title="Nodes" {...mapNodesToProps(nodes)} />
+    <InventoryRow title="Disks" {...diskStatsToProps(disks)} />
```

Comment:
not sure here, but please be consistent. Can we rename the prop to `diskStats` when we have a function `diskStatsToProps`? Or the other way around?

Reply:
Ack!